source/Threema/MsgApi/PublicKeyStore.php

```diff
@@ -55,6 +55,7 @@
 		$this->cache[$threemaId] = $publicKey;
 		return $this->savePublicKey($threemaId, $publicKey);
 	}
+    
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Whitespace found at end of line](https://www.codacy.com/app/c917250b/threema-msgapi-sdk-php/file/2194332756/issues/source?bid=2945997&fileBranchId=3167445#l58)